cannot build MsalAngularjsDemoApp

## I'm submitting a...
<!-- Check one of the following options with "x" -->
<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ x] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>


## Library version

<pre><code>
Library version: 1.0.2
<!-- Check whether this is still an issue in the most recent version -->
</code></pre>

## Current behavior
After performing 'npm install' and 'npm start' as descrobed in the README.md, I get this error:
> msal-angularjsapp@0.1.0 start C:\Users\itr00626\source\repos\microsoft-authentication-library-for-js\lib\msal-angularjs\samples\MsalAngularjsDemoApp
> webpack-dev-server

C:\Users\itr00626\source\repos\microsoft-authentication-library-for-js\lib\msal-angularjs\samples\MsalAngularjsDemoApp\node_modules\ajv-keywords\keywords\instanceof.js:52
    throw new Error('invalid "instanceof" keyword value ' + c);
    ^

Error: invalid "instanceof" keyword value Promise
    at getConstructor (C:\Users\itr00626\source\repos\microsoft-authentication-library-for-js\lib\msal-angularjs\samples\MsalAngularjsDemoApp\node_modules\ajv-keywords\keywords\instanceof.js:52:11)
    at Ajv.compile (C:\Users\itr00626\source\repos\microsoft-authentication-library-for-js\lib\msal-angularjs\samples\MsalAngularjsDemoApp\node_modules\ajv-keywords\keywords\instanceof.js:21:27)
    at Object.useCustomRule (C:\Users\itr00626\source\repos\microsoft-authentication-library-for-js\lib\msal-angularjs\samples\MsalAngularjsDemoApp\node_modules\ajv\lib\compile\index.js:289:26)
    at Object.generate_custom [as code] (C:\Users\itr00626\source\repos\microsoft-authentication-library-for-js\lib\msal-angularjs\samples\MsalAngularjsDemoApp\node_modules\ajv\lib\dotjs\custom.js:31:24)
    at Object.generate_validate [as validate] (C:\Users\itr00626\source\repos\microsoft-authentication-library-for-js\lib\msal-angularjs\samples\MsalAngularjsDemoApp\node_modules\ajv\lib\dotjs\validate.js:228:37)
    at Object.generate_anyOf [as code] (C:\Users\itr00626\source\repos\microsoft-authentication-library-for-js\lib\msal-angularjs\samples\MsalAngularjsDemoApp\node_modules\ajv\lib\dotjs\anyOf.js:34:27)
    at generate_validate (C:\Users\itr00626\source\repos\microsoft-authentication-library-for-js\lib\msal-angularjs\samples\MsalAngularjsDemoApp\node_modules\ajv\lib\dotjs\validate.js:228:37)
    at localCompile (C:\Users\itr00626\source\repos\microsoft-authentication-library-for-js\lib\msal-angularjs\samples\MsalAngularjsDemoApp\node_modules\ajv\lib\compile\index.js:99:22)
    at Ajv.compile (C:\Users\itr00626\source\repos\microsoft-authentication-library-for-js\lib\msal-angularjs\samples\MsalAngularjsDemoApp\node_modules\ajv\lib\compile\index.js:67:13)
    at _compile (C:\Users\itr00626\source\repos\microsoft-authentication-library-for-js\lib\msal-angularjs\samples\MsalAngularjsDemoApp\node_modules\ajv\lib\ajv.js:328:29)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! msal-angularjsapp@0.1.0 start: `webpack-dev-server`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the msal-angularjsapp@0.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\itr00626\AppData\Roaming\npm-cache\_logs\2019-07-03T06_45_44_297Z-debug.log


## Expected behavior
<!-- Describe what the desired behavior would be. -->
Start http server without errors.

--- a/lib/msal-angular/samples/MSALAngularDemoApp/package.json
+++ b/lib/msal-angular/samples/MSALAngularDemoApp/package.json
@@ -20,11 +20,11 @@
     "@angular/platform-browser": "^5.0.0",
     "@angular/platform-browser-dynamic": "^5.0.0",
     "@angular/router": "^5.0.0",
+    "@azure/msal-angular": "0.1.2",
     "core-js": "^2.4.1",
     "rxjs": "^5.1.0",
     "ts-helpers": "^1.1.1",
-    "zone.js": "^0.8.4",
-    "@azure/msal-angular": "file:../.."
+    "zone.js": "^0.8.4"
   },
   "devDependencies": {
     "@angular/cli": "^1.0.1",

--- a/lib/msal-angularjs/samples/MsalAngularjsDemoApp/package.json
+++ b/lib/msal-angularjs/samples/MsalAngularjsDemoApp/package.json
@@ -12,8 +12,9 @@
     "@azure/msal-angularjs": "0.1.1"
   },
   "devDependencies": {
-    "webpack": "^2.4.1",
-    "webpack-dev-server": "^3.1.11",
-    "angular": "^1.7.0"
+    "angular": "^1.7.0",
+    "webpack": "^4.35.3",
+    "webpack-cli": "^3.3.5",
+    "webpack-dev-server": "^3.7.2"
   }
 }

--- a/lib/msal-angularjs/samples/MsalAngularjsDemoApp/webpack.config.js
+++ b/lib/msal-angularjs/samples/MsalAngularjsDemoApp/webpack.config.js
@@ -1,6 +1,7 @@
 var path = require('path');
 
 module.exports = {
+  mode: "development",
   entry: path.resolve(__dirname, 'app.js'),
   output: {
     path: path.resolve(__dirname),

